src/components/card/event-card/index.js

```diff
@@ -75,18 +74,18 @@ const EventCard = (props: Props) => {
             <ShowDate date={date} color="gray" fontSize="12" />
             {
               isBookmarked
-                ? <BookmarkIcon type="bookmark" size="lg" color="gray" onClick={handleClickBookmark} />
-                : <BookmarkIcon type="bookmark-border" size="lg" color="gray" onClick={handleClickBookmark} />
```

Comment:
isBookmarked condition can be set only for type property cause the rest are the same

Reply:
U R right. I'll consider it.